core/src/test/scala/integration/kafka/api/SaslSslAdminClientIntegrationTest.scala

```diff
@@ -449,7 +449,8 @@ class SaslSslAdminClientIntegrationTest extends AdminClientIntegrationTest with
     validateMetadataAndConfigs(createResult)
     val createResponseConfig = createResult.config(topic1).get().entries.asScala
 
-    val describeResponseConfig = describeConfigs(topic1)
+    val topicResource = new ConfigResource(ConfigResource.Type.TOPIC, topic1)
```

Comment:
I'm not sure about this change here may need someone from core to review, otherwise lgtm. cc @hachikuji @soondenana

Reply:
Actually I don't think this change is necessary.